### PR TITLE
IGNITE-15686 TableExample and KeyValueBinaryViewExample fail with NPE

### PR DESCRIPTION
**Pull request by vldpyatkov**

### CODE REVIEW

File: examples/src/main/java/org/apache/ignite/example/table/KeyValueBinaryViewExample.java [hunk 1]
```diff
@@ -43,72 +46,71 @@
  */
 public class KeyValueBinaryViewExample {
     public static void main(String[] args) throws Exception {
-        Ignite ignite = IgnitionManager.start(
+        try (Ignite ignite = IgnitionManager.start(
             "node-0",
-            Files.readString(Path.of("config", "ignite-config.json").toAbsolutePath()),
+            Files.readString(Path.of("config", "ignite-config.json")),
```
Comment (isapego) [inline examples/src/main/java/org/apache/ignite/example/table/KeyValueBinaryViewExample.java:51]: In `SqlJdbcExample` we are using path `"examples/config/ignite-config.json"`. Need to unify it (change path here or there)

**Review by isapego — CHANGES_REQUESTED** [review]
Overall looks good, only one comment from me.

File: examples/src/main/java/org/apache/ignite/example/table/KeyValueBinaryViewExample.java (hunk 1 above)
Comment (vldpyatkov) [reply examples/src/main/java/org/apache/ignite/example/table/KeyValueBinaryViewExample.java:51]: Made it the same in SqlJdbcExample.

**Review by isapego — APPROVED** [review]
Looks good now

**Pull request closed without merging**